lib/components/dropdown-item.vue

```diff
@@ -1,31 +1,21 @@
 <template>
     <b-link class="dropdown-item"
             role="menuitem"
```

Comment:
I'll make a mockup in the playground or jsFiddle and try placing a regular button element within a dropdown menu (and adding the appropriate `dropdown-item` class and see how it works